config.md

```diff
@@ -147,6 +147,23 @@ flag parsing (clang vs clang-cl), target inference (gcc-arm-noneabi) etc.
 If the option matches a glob mentioned in `--query-driver`, then it'll be
 invoked for extraction of include paths.
 
+### BuiltinHeaders
+{:.v21}
```

Comment:
This is actually the first patch documenting something new in clangd 21 (thank you for being proactive!), so the style needs to be defined [here](https://github.com/llvm/clangd-www/blob/8e1c4c6869f03763ce2999339413d099e968b669/styles.css#L199) (and [this rule](https://github.com/llvm/clangd-www/blob/8e1c4c6869f03763ce2999339413d099e968b669/styles.css#L173) needs adjustment as well).